db/models.py

```diff
@@ -1 +1,41 @@
 from django.db import models
+
+
+class Race(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(blank=True)
+
+
+class Skill(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    bonus = models.CharField(max_length=255)
+    race = models.ForeignKey(
+        Race,
+        on_delete=models.SET_NULL,
+        null=True
+    )
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/225003380-9250de8e-e14d-45f2-b597-8539d9abc4ac.png)
are u sure that skill can have no race ?